blank-canvas-blocks/sass/blocks/_group.scss

```diff
@@ -1,7 +1,9 @@
-.wp-block-group {
-	&.has-background {
-		.wp-block-group__inner-container {
-			padding: var(--wp--custom--margin--vertical) var(--wp--custom--margin--horizontal);
-		}
+// NOTE: Gutenberg sets a padding value by default for groups with a background.
+// However when this top level element has padding applied it mucks up the alignment
+// calculations, so it's removed and set instead (to a configurable value) on the inner-container instead.
+.wp-block-group.has-background {
+	padding: unset;
+	.wp-block-group__inner-container {
+		padding: var(--wp--custom--margin--vertical) var(--wp--custom--margin--horizontal);
```

Comment:
> Any blocks, inside of a group, set to alignwide should be as wide as it's container, however wide that is.

What about nested blocks that do not have an alignment set? Right now these are defaulting to the width of the container block, less the padding. IMO they should be the default width, meaning we may need a rule like this: 

```scss
	.wp-block-group__inner-container {
		*:not(.alignwide):not(.alignfull) {
			max-width: var(--wp--custom--width--default);
			margin-left: auto;
			margin-right: auto;
		}
	}
```

Reply:
I kind of went back and forth about this one and wound up thinking otherwise (though I've gone back-and-forth).
[This](https://github.com/Automattic/themes/pull/3449/files#diff-24c1332a2754d3bffa38ace74a2f607b1c1a1b7fb3fc2cebb951ecea826cf811R39) ensures that if you nest GROUPS then that's the behavior so there's an avenue for that behavior.  But I kind of thought that if I made a group and made it full width, then dropped a paragraph in it then I would expect it to follow the width of the group I just added to it.  

I (mostly) don't disagree with you though, and to me that feels like behavior (whatever it is) that should be consistent across Themes and not opinionated.  How does that compare to how the incoming alignment changes are expected to default to?  I suggest standardizing on whatever behavior that is.